src/frontend/src/styles/global.css

```diff
@@ -2,10 +2,3 @@
     background: #007f3b;
     color: #ffffff;
 }
-
-.app-footer {
-    background-color: #d8dde0;
-    border-top: 4px solid #005eb8;
-    padding-bottom: 24px;
-    padding-top: 24px
-}
```

Comment:
I guess this answers my above question r.e. the footer. But how come it's removed in this ticket?

Reply:
See above, no longer needed. Styling looks as intended without it